FlowCrypt/Functionality/Services/Backup Services/BackupService.swift

```diff
@@ -74,10 +74,11 @@ extension BackupService: BackupServiceType {
                 atts: attachments,
                 pubKeys: nil
             )
-            let backupEmail = try self.core.composeEmail(msg: message, fmt: .plain, pubKeys: message.pubKeys)
 
-            self.messageSender
-                .sendMail(mime: backupEmail.mimeEncoded)
+            self.core.composeEmail(msg: message, fmt: .plain, pubKeys: message.pubKeys)
+                .map(\.mimeEncoded)
+                
+                .flatMap(self.messageSender.sendMail)
                 .sink(
                     receiveCompletion: { result in
                         switch result {
```

Comment:
A bit of a mess here combining `Promise` and `Future` but not your fault, should be cleaned up in #404

Reply:
Maybe we could have a separate PR later to covert this `backupToInbox` to a Future. I'll file it.